gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -132,6 +133,19 @@ class BaselineIdea extends AbstractBaselinePlugin {
     }
 
     /**
+     * Configure the default working directory of JUnit tests to be the module directory.
+     */
+    private void addJUnitWorkingDirectory(node) {
+        node.append(new XmlParser().parseText('''
+            <component name="RunManager">
+                <configuration default="true" type="JUnit" factoryName="JUnit">
```

Comment:
cheers

On Sun, Mar 20, 2016 at 6:38 PM nmiyake <notifications@github.com> wrote:

> In
> gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy
> <https://github.com/palantir/gradle-baseline/pull/45#discussion_r56777448>
> :
>
> > @@ -132,6 +133,19 @@ class BaselineIdea extends AbstractBaselinePlugin {
> >      }
> >
> >      /**
> > +     * Configure the default working directory of JUnit tests to be the module directory.
> > +     */
> > +    private void addJUnitWorkingDirectory(node) {
> > +        node.append(new XmlParser().parseText('''
> > +            <component name="RunManager">
> > +                <configuration default="true" type="JUnit" factoryName="JUnit">
>
> Good call -- if we do it for debug, would make sense for the Application
> configuration as well. Will add that.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/palantir/gradle-baseline/pull/45/files/38f2e0d21f66c6c27d0f166f94a93282bd7838c9#r56777448>
>